lib/src/migrators/module.dart

```diff
@@ -496,7 +546,8 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
     var configured = <String>[];
     for (var name in locallyConfiguredVariables.keys) {
       var variable = locallyConfiguredVariables[name];
-      if (variable.isGuarded || _referencedVariables.contains(variable)) {
+      if (variable.isGuarded ||
+          references.variables.keysForValue(variable).isNotEmpty) {
```

Comment:
```suggestion
          references.variables.containsValue(variable)) {
```

This seems like a generally useful method for `BidirectionalMap`.